chart/infra-server/static/flavors.yaml

```diff
@@ -255,11 +258,23 @@
       value: ocp/stable
       kind: optional
       help: |
-        The latest stable Red Hat enterprise OCP releases can be found at https://mirror.openshift.com/pub/openshift-v4/clients/ocp/.
-        For officially released OCP versions, use the path to one of the stable releases, e.g., ocp/stable-4.
-        Dev preview Red Hat enterprise OCP builds can be found at https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp-dev-preview/.
-        For dev preview OCP versions, use the path to one of the dev preview releases, e.g., ocp-dev-preview/latest.
-        Opensource OKD releases are at https://github.com/openshift/okd/releases.
+        ## Stable OCP
+
+        The latest stable Red Hat enterprise OCP releases can be found at
+        [clients/ocp/](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/). For
```

Comment:
Is the wrapping by prettier for markdown?

Especially if output respects line breaks
* **For** at beginning of a new line might help scanning.
* It might also remove the need for final period after the link.